### PR TITLE
Replace `regexp` with `strings`

### PR DESCRIPTION
**Pull request by tamird**

### CODE REVIEW

File: storage/client_split_test.go [hunk 1]
```diff
@@ -132,10 +132,10 @@ func TestStoreRangeSplitConcurrent(t *testing.T) {
 			args, reply := adminSplitArgs(proto.KeyMin, []byte("a"), 1, store.StoreID())
 			err := store.ExecuteCmd(context.Background(), proto.Call{Args: args, Reply: reply})
 			if err != nil {
-				if matched, regexpErr := regexp.MatchString("range is already split at key", err.Error()); !matched || regexpErr != nil {
-					t.Errorf("error %s didn't match regex %v", err, regexpErr)
-				} else {
+				if strings.Contains(err.Error(), "range is already split at key") {
```
Comment (petermattis) [inline storage/client_split_test.go]: `testutils.IsError`?

File: util/caller/resolver_test.go [hunk 2]
```diff
@@ -41,7 +42,7 @@ func TestCallResolver(t *testing.T) {
 			return cr.Lookup(1)
 		}()
 		fileline := fmt.Sprintf("%s:%d", file, line)
-		if matched, err := regexp.MatchString(`^resolver_test\.go`, fileline); !matched || err != nil {
+		if !strings.HasPrefix(fileline, "resolver_test.go") {
```
Comment (petermattis) [inline util/caller/resolver_test.go]: Ditto: `testutils.IsError`?

File: storage/client_split_test.go (hunk 1 above)
Comment (tamird) [inline storage/client_split_test.go]: I did that at first, but it's redundant with the nil check above

File: util/caller/resolver_test.go (hunk 2 above)
Comment (tschottdorf) [inline util/caller/resolver_test.go]: actually, `!strings.Equal`

Comment (tamird) [inline util/caller/resolver_test.go]: @petermattis there is no error here

Comment (tamird) [inline util/caller/resolver_test.go]: > actually, !strings.Equal

Nope, there's a line number on the end

Comment (petermattis) [inline util/caller/resolver_test.go]: Oh, oops. Distracted while code reviewing.

Comment (tschottdorf) [inline util/caller/resolver_test.go]: > Nope, there's a line number on the end

yeah, but we don't match it, so just test `line` and drop `fileline`.

Comment (tamird) [inline util/caller/resolver_test.go]: fixed here and above

**Comment by tschottdorf** [comment]
LGTM

**Comment by tamird** [comment]
Replacing `path` with `filepath` is blocked on `https://github.com/jteeuwen/go-bindata/pull/99/files`. Will open a new PR when that's merged.

**Pull request merged**